### ISSUE TITLE
MCF generated with mcfgen has doubled fragment connection.

### ISSUE THREAD

**Issue by hmcezar** [MaginnGroup/Cassandra#83]
## Expected behavior
If fragments 1 and 2 are connected, only one connection should appear in the MCF file.

## Current behavior
The connection is doubled.
```
# Fragment_Connectivity
2
1    1    2
2    2    1
```

## Steps to reproduce
1. Run `mcfgen.py C4H10.pdb` in the `Methane_Butane` example.

## Additional context
The old Python 2 version of mcfgen.py distributed with V1.2 had the correct behavior:
```
$ diff C4H10.mcf v1.2_C4H10.mcf 
68c68
< 2
---
> 1
70d69
< 2    2    1
```

**Comment by rsdefever** [comment]
@hmcezar could you checkout PR #84 and make sure everything is working on your end. I ran it through several tests but want to make sure everything is fixed before merging/patch release.

It appears an older version of `mcfgen.py` was inadvertently merged into the main code branch when we were performing some repo cleaning earlier this spring. The main code has integration tests that ensure that any changes don't affect the results of the code. Unfortunately our auxiliary scripts (`mcfgen.py` and `library_setup.py`) are not yet covered by such tests. If they were this error would have been caught immediately.

I will also point you towards this package [here](https://mosdef-cassandra.readthedocs.io/en/latest/). Depending on your simulation workflow you may find it to be a cleaner interface. That is the primary way I use the Cassandra code now (full disclosure: I wrote the package, so I'm a bit biased :)).

**Comment by hmcezar** [comment]
@rsdefever I've checked the new version in some zeolites and nanotubes (rigid frameworks) and everything seems to be working fine.

Concerning molecules, I tested only with butane from the `Methane_Butane` directory. Besides some stylistic differences in the generated `.ff`, everything seems to be fine. If I use the '.ff` from the example, the final `.mcf` is consistent with the from the example (the only difference being the atom names).
If you guys plan to introduce some testing for this part, be sure to rerun the examples first to generate new files, because as I said, there are some differences in the style between the current files and the ones generated with the new `mcfgen.py`.

Thanks for taking the time to fix the tool!

Concerning MoSDeF Cassandra, I'll definitely check it out! Thanks for pointing it out.
